checksum/crc/crc8.go

```diff
@@ -0,0 +1,53 @@
+// crc8.go
+// description: Calculate CRC8
+// details:
+// A cyclic redundancy check (CRC) is an error-detecting code commonly used in digital networks and storage devices to detect accidental changes to raw data. See more [CRC](https://en.wikipedia.org/wiki/Cyclic_redundancy_check)
```

Comment:
Please make this a multi line comment, it gets difficult to read on small screens.